source/PlayState.hx

```diff
@@ -1,12 +1,18 @@
 package;
 
+import flixel.FlxSprite;
 import flixel.FlxState;
 
 class PlayState extends FlxState
 {
+	var hero:FlxSprite;
```

Comment:
Instead of declaring this with the FlxSprite type, we want to declare (and later instantiate)  with our newly created Hero class as the type.